portugues/cogumelos/cogumelos.md

```diff
@@ -0,0 +1 @@
+Você tem uma onda muito louca e andando pela terra do nunca e lutando com o capital gancho você tem a idéia para consertar o erro do bug.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.